engine/app/components/citizens_advice_components/footer.html.erb

```diff
@@ -55,6 +55,11 @@
         <p class="cads-footer__meta-text" data-testid="copyright"><%= t("citizens_advice_components.footer.copyright", year: current_year) %></p>
         <p class="cads-footer__meta-text" data-testid="legal-summary"><%= legal_summary_text %></p>
       </div>
+      <% if additional_logo.present? %>
+        <div class="cads-footer__additional-logo">
+          <img src="<%= additional_logo.src %>" alt="<%= additional_logo.alt %>">
```

Comment:
I'm a little wary of the `img` tag here. For the similar custom logo behaviour in the header we allow passing a plain slot instead (we typically render an inline SVG but it gives the consumer the choice then).